packages/protocol/dev-scripts/delete-flattened-licenses.js

```diff
@@ -0,0 +1,9 @@
+const replace = require('replace-in-file');
+
+const results = replace.sync({
+  files: '*.flat.sol',
+  from: /.*SPDX-License-Identifier.*\n/g,
```

Comment:
Why are we removing this "license-identifier"? I'm just curious

Reply:
I don't know :)

Reply:
But things break without removing it?

Reply:
No, I guess this is unrelated to core development. Maybe if you want to copy and modify the contract? Not really sure, I think @bbenligiray would know the answer.